rs-matter/src/data_model/objects/encoder.rs

```diff
@@ -160,7 +160,10 @@ impl<'a, 'b, 'c> AttrDataEncoder<'a, 'b, 'c> {
         };
 
         if let Some(status) = status {
-            AttrResp::Status(status).to_tlv(tw, TagType::Anonymous)?;
+            let resp = AttrResp::Status(status);
+            warn!("Read status: {:?}", resp);
```

Comment:
Prefer clearer language like "Unsuccessful read status: " and "Unsuccessful write status: " -- otherwise it's easy to misinterpret as general `info!` level logging.